pipeline/experiment.py

```diff
@@ -195,7 +202,7 @@ class SessionComment(dj.Manual):
 
 @schema
 class SessionDetails(dj.Manual):
-    # TODO: better table name
+    # TODO: more specific table name?
```

Comment:
How about something like `SessionSubjectInfo` or `SessionSubjectData`?

Reply:
I don't think we have to include the Subject in the table name. A session can belong to only one subject, so everything related to a session is related to the subject, too.
Is there a rule for table names? It should definitely be short and specific.
But I agree with Info or Data, or MetaData.

Reply:
This table contains information about the weight and the water management for the animal per session, then how about `SessionWaterDetails` or `SessionWaterInfo`. 
Unless we plan on adding other non-water related, I think just `SessionInfo` is too general

Reply:
I agree. But I think there might be other session related metadata we might want to record in the future. What about adding a more general table like SessionDetails/SessionMetadata and add SessionWaterDetails as a part table so it is extendable in the future with more part tables.